src/pages/login/login.ts

```diff
@@ -21,38 +21,31 @@ export class LoginPage {
   user      : User;
 
   constructor(public navCtrl: NavController,
-              public navParams: NavParams,
```

Comment:
If you remove this, you must remove the import too.
'NavParams' is declared but never used.